setenv.sh

```diff
@@ -1,91 +1,54 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
Since bash is used here, ZOPEN_DEPS should as bash as a dependency

Reply:
fwiw, that is just a script for doing resumption work outside of `zopen build`.  ie. when a build is complete and you want to experimient/build v8base/v8 bypassing the work done by `zopen build` to create venv, activate, fetch, etc and just run `gn gen` and `ninja` on the result.

Never the less, I updated bash as a dependency.